core/src/main/scala/package.scala

```diff
@@ -48,6 +48,7 @@ package object funnel {
     def append(a: A, b: => A) = G.plus(a, b)
   }
 
-  implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
-  implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
+  // This doesn't work in 2.11; these are at the highest priority within the package.
+  // implicit val dontUseTheDefaultStrategy: scalaz.concurrent.Strategy = null
+  // implicit val theDefaultStrategyCausesProblems: scalaz.concurrent.Strategy = null
 }
```

Comment:
These bindings are at the highest priority in 2.11 which means it is not possible to use an implicit `Strategy` anywhere in package `funnel`. So they need to go somewhere else I think.